internal/contour/route.go

```diff
@@ -189,6 +195,17 @@ func (v *routeVisitor) visit(vertex dag.Vertex) {
 	}
 }
 
+func mergePathPrefixes(conditions []dag.Condition) string {
+	mergedPath := ""
```

Comment:
I'd suggest start with "/"

Reply:
In my previous impl I had the builder adding a `/` to the prefix if it wasn't defined. This way you're not required to add a `Condition` field to the route if you don't want to (e.g. you have an `Include` giving you the prefix. I could put that logic back if that makes sense to you.

The other thing I was doing was using the absence of a prefix to flag that we're doing some other type of routing (e.g. regex). If `mergePathPrefixes` now returns `/` all the time, I'm not sure how to determine if it's regex or not.

Reply:
I'll put the quick regex check in and move along. =)